### PR TITLE
EE-1217: add initial support for `get_call_stack`

### PR DESCRIPTION
**Pull request by henrytill**

### CODE REVIEW

File: execution_engine_testing/tests/src/test/regression/mod.rs [hunk 1]
```diff
@@ -24,7 +25,6 @@ mod ee_550;
 mod ee_572;
 mod ee_584;
 mod ee_597;
-mod ee_598;
```
Comment (henrytill) [inline execution_engine_testing/tests/src/test/regression/mod.rs:27]: Note that we've disabled the EE-598 regression for the moment, as it uses proxy contract in a way that no longer works with the current checks.  

A fix will follow...

**Comment by dwerner** [comment]
This is passing the upgrade test:

https://gist.github.com/dwerner/eade3ce34f71788c8122b0b66c88cba2

**Review by EdHastingsCasperLabs — APPROVED** [review]

**Comment by goral09** [comment]
Hey Henry, can you include the description of that new function? Or link a GH/ZenHub issue that describes it?

**Comment by henrytill** [comment]
Converting back to a draft after realizing it would be more useful to include contract hash rather than contract wasm hash in the `Contract` variant.

**Comment by henrytill** [comment]
closed in favor of #1627

**Pull request closed without merging**